VendingMachine/UI/Tooltip.cs

```diff
@@ -0,0 +1,106 @@
+using Godot;
+
+namespace untitledplantgame.VendingMachine;
+
+public partial class Tooltip : Control
```

Comment:
Name könnte klarer sein (nur für Gesichter; kein allgemeiner Tooltip)

Reply:
Changed to EmojiTooltip